routing/localchans/manager.go

```diff
@@ -40,6 +49,11 @@ type Manager struct {
 	FetchChannel func(tx kvdb.RTx, chanPoint wire.OutPoint) (
 		*channeldb.OpenChannel, error)
 
+	// AddEdge is used to add edge/channel to the topology of the router,
+	// after all information about channel will be gathered this
```

Comment:
Nit: not really sure what you mean by this comment: `after all information about channel will be gathered this`

Reply:
I copied the comment from the router. I'll drop that part, because it doesn't make sense to me either.

Reply:
Done